src/objects/nodes.ts

```diff
@@ -117,6 +149,7 @@ function createContainer(): Container {
   const container = new Container()
 
   container.eventMode = 'none'
+  container.visible = false
```

Comment:
Noteworthy that rather than waiting to add the node container objects to pixi we're adding them but setting their visibility to `false`. Once the layout comes back from the worker and all the node positions are updated then their visibility is set to `true`.